data-serving/data-service/schemas/cases.schema.json

```diff
@@ -204,19 +204,20 @@
           "_id": {
             "bsonType": "objectId"
           },
-          "provided": {
+          "values": {
             "bsonType": "array",
             "uniqueItems": true,
             "items": {
               "bsonType": "string"
             }
           },
-          "imputed": {
-            "bsonType": "array",
-            "uniqueItems": true,
-            "items": {
-              "bsonType": "string"
-            }
+          "status": {
+            "enum": [
+              "Asymptomatic",
+              "Presymptomatic",
+              "Symptomatic",
+              "Unknown"
```

Comment:
Do we want Unknown as a specific value? Alternative would be to not populate this field. I think we've gone in the latter direction for other fields.

Reply:
I believe Stephen wants to make these fields mandatory with Unknown as the default, so there's less room for ambiguity -- there was some back and forth on the spec about this: https://docs.google.com/spreadsheets/d/1qOnGJQg1gtbmNxAW8c9hC1TvFpIQmSnORD-x-iXrpJ0/edit?disco=AAAAJn4cUhc